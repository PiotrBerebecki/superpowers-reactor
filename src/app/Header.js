import React from 'react';
import { NavMenuLink } from './../styled';

function Header() {
  return (
    <div>
      <NavMenuLink exact to="/latest">latest</NavMenuLink>
      <NavMenuLink exact to="/trending">trending</NavMenuLink>
      <NavMenuLink exact to="/top">top</NavMenuLink>
    </div>
  );
}

export default Header;
