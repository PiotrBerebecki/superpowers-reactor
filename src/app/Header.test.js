import React from 'react';
import renderer from 'react-test-renderer';
import { shallow } from 'enzyme';
import { MemoryRouter } from 'react-router-dom';

import Header from './Header';

describe('Header', () => {
  test('renders without crashing', () => {
    shallow(<Header />);
  });

  // test('renders without crashing', () => {
  //   const component = shallow(<Header />);
  // });

  test('renders correctly', () => {
    const tree = renderer
      .create(<MemoryRouter><Header /></MemoryRouter>)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
