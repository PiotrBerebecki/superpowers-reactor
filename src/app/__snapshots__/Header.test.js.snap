// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header renders correctly 1`] = `
<div>
  <a
    className="sc-bdVaJa gRwgNw"
    href="/latest"
    onClick={[Function]}
    style={undefined}
  >
    latest
  </a>
  <a
    className="sc-bdVaJa gRwgNw"
    href="/trending"
    onClick={[Function]}
    style={undefined}
  >
    trending
  </a>
  <a
    className="sc-bdVaJa gRwgNw"
    href="/top"
    onClick={[Function]}
    style={undefined}
  >
    top
  </a>
</div>
`;
